xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.cpp

```diff
@@ -41,7 +34,14 @@ bool CDVDInputStream::Open(const char* strFile, const std::string &content, bool
   // but keep them in m_url.
   url.SetProtocolOptions("");
   m_strFileName = url.Get();
+}
 
+CDVDInputStream::~CDVDInputStream()
+{
+}
+
+bool CDVDInputStream::Open(const std::string &content, bool contentLookup)
```

Comment:
could you remove content and contentLookup too? they are also passed in CreateInpuStream